docs/hypernode-platform/autoscaling/how-does-horizontal-autoscaling-work.md

```diff
@@ -63,44 +63,41 @@ We will divide them between Hypernode-specific and Application-specific requirem
 
 #### Operating system
 
-- The operating system of the Hypernode should be Debian Bookworm. If you would like to upgrade the os of your Hypernode, feel free to contact our support team for help. https://www.hypernode.com/en/contact/
+Horizontal autoscaling is only available on Hypernodes with the Debian Bookworm operating system.
+
+If you would like to upgrade the operating system of your Hypernode, feel free to [contact our support team](https://www.hypernode.com/en/contact/) for help.
 
 #### Make sure the Hypernode is a production plan
 
 For now, we don't support horizontal autoscaling for development plans.
 
 #### Enable and configure Varnish
 
-To make use of Horizontal autoscaling, Varnish should be enabled and configured on the Hypernode.
-You can check if Varnish is enabled on your Hypernode by running
+Varnish is a requirement, because it's essential to deliver cached pages from the load balancer instead of the application servers when scaled up.
+The benefits are:
 
-```console
-hypernode-systemctl settings varnish_enabled
-```
+- Pages in the cache are served faster, because it is one less network hop
```

Comment:
maybe good to refer to https://docs.hypernode.com/hypernode-platform/varnish/how-to-enable-varnish-on-hypernode.html in case they don't have it enabled?

Reply:
We already do that later in this section